spec/features/user_adds_new_bookstore_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+feature "user adds a new bookstore page" do
+  scenario 'user successfully adds a bookstore' do
+    visit '/'
+
+    click_link "Add New Bookstore"
+
+    fill_in "Name", with: "Booky"
+    fill_in "Address", with: "102 Main Street"
+    fill_in "City", with: "Cambridge"
+    select "Massachusetts", from: "State"
+    fill_in "Zip Code", with: "99999"
+    click_button "Add Bookstore"
+
+    expect(page).to have_content "Booky Reviews"
+  end
+
+  scenario 'user tries to submit a blank form' do
+    visit '/'
+
+    click_link "Add New Bookstore"
+
+    click_button "Add Bookstore"
+
+    expect(page).to have_content ("
```

Comment:
You don't need to check for everything that you expect to be wrong. It's fine to test for just a couple of these messages.

So for example, you could do something like:
```
expect(page).to have_content "Name can't be blank"
expect(page).to have_content "Zip code is not a number"
```